sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.2.0 (2020-01)
+- Bumped to service version 2019-07-07.
```

Comment:
Nit - should all of these be `Added support for service version ...` because we're supporting both and not moving unilaterally?

Reply:
Fixed all instances.